web/l10n/de.json

```diff
@@ -144,6 +144,8 @@
     "commandData": "Daten",
     "commandPhone": "Rufnummer",
     "commandMessage": "Nachricht",
+    "commandTimezoneOffset": "Zeitverschiebung zu UTC",
```

Comment:
okay ... also the english one? But in en.json it's fine, right? Shall I rebase or just revert the file?